### DISCUSSION TITLE
Video Texture on WebWorker

### DISCUSSION THREAD

**Discussion by aidevelopercode** [pixijs/pixijs#10724] [v8 General]
Hello, 

i try to load video and play it into pixijs offScreenCanvas on a webworker. 
But the porbleme is the video could not be loaded using Assets.load('video.mp4') caused by document not found on web worker. 

Any Help please ? 
Thank you

**Comment by bigtimebuddy** [comment]
Video is not supported in webworkers because it uses `document.createElement('video')` to create the video element.
